icu4c/source/i18n/ucol.cpp

```diff
@@ -102,6 +102,24 @@ ucol_safeClone(const UCollator *coll, void * /*stackBuffer*/, int32_t * pBufferS
     return newColl->toUCollator();
 }
 
+U_CAPI UCollator* U_EXPORT2
+ucol_clone(const UCollator *coll, UErrorCode *status)
+{
```

Comment:
The code is slightly duplicated, because it needs to avoid emitting U_SAFECLONE_ALLOCATED_WARNING. It's the consistent with ubrk_clone introduced last year.

Reply:
Please make the same changes here -- make the old function not set the warning when it's not relevant, with an equivalent small change to the docs.

Reply:
Done.